test/utils/shippable/shippable.sh

```diff
@@ -17,6 +17,15 @@ for container in $(docker ps --format '{{.Image}} {{.ID}}' | grep -v '^drydock/'
     docker rm -f "${container}" || true  # ignore errors
 done
 
+for i in network windows sanity; do
+    if [[ "${script}" == "${i}" ]]; then
+        # Prefetch the default test container to avoid any
+        # slow down during the test execution
+        docker pull quay.io/ansible/default-test-container
```

Comment:
This will fetch the latest version of the image. We need to get the correct version from the `docker.txt` file.

Reply:
Arg, indeed.